### ISSUE TITLE
PDF view doesn't support mouse back/forward buttons

### ISSUE THREAD

**Issue by AbeJellinek** [zotero/zotero#3436]
https://forums.zotero.org/discussion/108019/bug-alt-left-right-arrow-doesnt-go-back-forward-in-epub-and-pdf-reader/p1

Mozilla calls the buttons "Browser Back" and "Browser Forward": https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button

**Comment by mrtcode** [comment]
https://forums.zotero.org/discussion/109225/feature-request-add-a-shortcut-about-back-button

**Comment by mabynke** [comment]
Is there any progress on this? I am really missing this functionality.

**Comment by Tianmaru** [comment]
Although I believe the mouse back/forward buttons were working in Zotero 6, after updating to Zotero 7.0.3, they don't appear to be bound to the reader's "Back" action anymore.

**Comment by ziyuanding** [comment]
> Although I believe the mouse back/forward buttons were working in Zotero 6, after updating to Zotero 7.0.3, they don't appear to be bound to the reader's "Back" action anymore.

Just tried in Zotero 6.0.36 with no luck..

A (tempory) solution for logitech mouse users would be mapping to keyboard shortcuts: 
In logi options+, replace your forward side button's mapping from "forward" to "Alt + Right Arrow", replace your backward side button's mapping from "backward" to "Alt + Left Arrow". As 4th and 5th mouse buttons events are not working in zotero right now.

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/121342/feature-request-customizable-navigation-shortcuts-and-mouse-side-button-support#latest

**Comment by mrtcode** [comment]
It likely needs to use mouse 3 and 4 buttons instead.

**Comment by davidruda** [comment]
Yes, this feature is very much needed. For the time being, I'm using a workaround described here: https://www.reddit.com/r/zotero/comments/1lprd7x/using_previousnext_mouse_buttons_in_pdf_reader/
